compiler/tests/pil.rs

```diff
@@ -61,6 +61,12 @@ fn test_witness_lookup() {
     );
 }
 
+#[test]
+#[should_panic]
```

Comment:
Using `#[should_panic]` will further the issue #312, i.e. we don't know if the thing failed where it was supposed to fail. That said, we would need a major refactor to make `verify_pil()` return error, so maybe leave as is or use `std::panic::catch_unwind()` and assert against the error message.